src/seedu/addressbook/commands/Command.java

```diff
@@ -35,6 +35,17 @@ protected Command() {
     public static String getMessageForPersonListShownSummary(List<? extends ReadOnlyPerson> personsDisplayed) {
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
+    /**
```

Comment:
Good documentation before the function implementation to have an overview what parameters will be taken by the function, and what the function should do.